internal/device/darwin.go

```diff
@@ -25,14 +27,33 @@ func GetPower() string {
 	return out.String()
 }
 
-func Notify() {
+func CheckPower() {
 	regex := regexp.MustCompile(POWER_REGEX)
```

Comment:
I try to avoid regexes wherever possible (they're terribly slow and expensive in Go, and we're almost always better off using Go itself to do the parsing). If we must use them, though, it's a neat trick to compile them outside the function, at package level. Then _executing_ the compiled expression is relatively fast.

Reply:
Moved it to package level in 49935f032987d302fb399aea495996eace55ddf2. Also, I will check if I can parse it using Go without regex